cosmos/honeywood.json

```diff
@@ -0,0 +1,61 @@
+{
+  "chainId": "HoneyWood",
```

Comment:
@HeesungB 
Hello, 
I saw that my pipelines are not passing and it is due to the fact that the chainId should be in lower case. But the chainID looks exactly like this 'HoneyWood' in CamelCase